app/news/newsItem.module.css

```
@@ -0,0 +1,22 @@
+:global(.newsItem) {
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)